### PR TITLE
misc: create a random id on generated plain object

### PR DESCRIPTION
**Pull request by ansmonjol**
Create a random ID on a plain object created “on the flight”, to ease some FE logic

### CODE REVIEW

File: app/graphql/types/customers/usage/charge.rb [hunk 1]
```diff
@@ -8,13 +8,18 @@ class Charge < Types::BaseObject
 
         field :amount_cents, GraphQL::Types::BigInt, null: false
         field :events_count, Integer, null: false
+        field :id, ID, null: false
```
Comment (ivannovosad) [inline app/graphql/types/customers/usage/charge.rb:11]: @ansmonjol Can we add the new field to the [spec](https://github.com/getlago/lago-api/blob/main/spec/graphql/types/customers/usage/charge_spec.rb#L8) as well, please?

**Review by ivannovosad — CHANGES_REQUESTED** [review]

File: app/graphql/types/customers/usage/charge.rb (hunk 1 above)
Comment (ivannovosad) [reply app/graphql/types/customers/usage/charge.rb:11]: I just added it.

**Review by ivannovosad — APPROVED** [review]

**Review by lovrocolic — APPROVED** [review]

File: app/graphql/types/customers/usage/charge.rb [hunk 2]
```diff
@@ -8,13 +8,18 @@ class Charge < Types::BaseObject
...
         field :units, GraphQL::Types::Float, null: false
 
         field :billable_metric, Types::BillableMetrics::Object, null: false
         field :charge, Types::Charges::Object, null: false
         field :filters, [Types::Customers::Usage::ChargeFilter], null: true
         field :grouped_usage, [Types::Customers::Usage::GroupedUsage], null: false
 
+        def id
+          SecureRandom.uuid
```
Comment (nudded) [inline app/graphql/types/customers/usage/charge.rb:20]: I think just `Random.uuid` would be good enough here? SecureRandom has some overhead to make sure it's secure, is that needed for this use-case?

File: app/graphql/types/customers/usage/charge.rb (hunk 1 above)
Comment (ansmonjol) [reply app/graphql/types/customers/usage/charge.rb:11]: Thanks a lot!!

File: app/graphql/types/customers/usage/charge.rb (hunk 2 above)
Comment (ansmonjol) [reply app/graphql/types/customers/usage/charge.rb:20]: Well, I just need a unique ID for each items, it does not matter if the ID changes every time I fetch the data.

Sounds like `Random.uuid` can potentially create twice or more the same ID?
This object is received in an array with potentially many others.

Comment (nudded) [reply app/graphql/types/customers/usage/charge.rb:20]: it's not possible for Random.uuid to generate the same ID. However, my concerns were based on outdated information and it shouldn't matter 👍

**Pull request merged**